tests/CoreFX/CoreFX.issues.json

```diff
@@ -670,6 +670,26 @@
             "namespaces": null,
             "classes": null,
             "methods": [
+                {
+                    "name" : "System.Numerics.Tests.Matrix4x4Tests.Matrix4x4LerpTest",
+                    "reason" : "https://github.com/dotnet/coreclr/pull/20994"
```

Comment:
Yes, because they are using the legacy helpers.

I'm not logging new issues because of https://github.com/dotnet/coreclr/pull/20920#issuecomment-437535193. Basically, CoreFX PR tests are currently running against a stale copy, once the Arcade migration completes (or maybe is further along), we will be running against a live copy and will start with an empty CoreFX.issues.json